src/main/resources/assets/styles/_cookieBanner.scss

```diff
@@ -0,0 +1,138 @@
+$font-roboto: Roboto, sans-serif;
```

Comment:
If these variables are placed outside of the `.cookie-banner`, then it can be accessed by other `.css` files. We should also use `ssb-white` and `ssb-green-4` directly instead of using a variable that is a reference to something that's already a variable.